package.json

```diff
@@ -24,21 +83,22 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-router-dom": "^5.1.7",
+    "@types/react-test-renderer": "^17.0.1",
     "@typescript-eslint/eslint-plugin": "^4.28.2",
     "@typescript-eslint/parser": "^4.28.2",
     "cypress": "^7.6.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-react": "^7.24.0",
-    "prettier": "^2.3.2"
+    "prettier": "^2.3.2",
+    "react-test-renderer": "^17.0.2"
   },
   "scripts": {
-    "build": "react-scripts build",
+    "build": "node scripts/build.js",
     "ci": "yarn install --frozen-lockfile",
-    "eject": "react-scripts eject",
-    "start": "react-scripts start",
-    "test": "react-scripts test",
+    "start": "node scripts/start.js",
+    "test": "node scripts/test.js",
```

Comment:
Why node instead of react-scripts? Is needed for loading the canary?

Reply:
From `yarn run eject`.